### PR TITLE
modified node number of each file. add string-interacting.md

### PR DESCRIPTION
**Pull request by benluo**

### CODE REVIEW

File: _zh-cn/overviews/scala3-book/string-interpolation.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
+---
+title: 字符串插值
+type: chapter
+description: This page provides more information about creating strings and using 字符串插值.
+languages: zh-cn
+num: 18
+previous-page: first-look-at-types
+next-page: control-structures
+redirect_from:
+  - /overviews/core/string-interpolation.html
```
Comment (bishabosha) [inline _zh-cn/overviews/scala3-book/string-interpolation.md:10]: you should also remove the file `_zh-cn/overviews/core/string-interpolation.md`
```suggestion
  - /zh-cn/overviews/core/string-interpolation.html
```

**Comment by benluo** [comment]
For the Links > Internal check, the following failures were found:

* At ./_site/zh-cn/scala3/book/string-interpolation.html:2588:

  internally linking to /zh-cn/zh-cn/scala3/book/string-interpolation.html, which does not exist


don't know why?

File: _zh-cn/overviews/scala3-book/fp-functions-are-values.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ title: 函数是值
 type: section
 description: This section looks at the use of functions as values in functional programming.
 language: zh-cn
-num: 44
-previous-page: fp-pure-functions
+num: 45
+previors-page: fp-pure-functions
```
Comment (jxnu-liguobin) [inline _zh-cn/overviews/scala3-book/fp-functions-are-values.md:7]: previous

**Comment by jxnu-liguobin** [comment]
> For the Links > Internal check, the following failures were found:
> 
> * At ./_site/zh-cn/scala3/book/string-interpolation.html:2588:
>   internally linking to /zh-cn/zh-cn/scala3/book/string-interpolation.html, which does not exist
> 
> don't know why?

try to remove `/zh-cn/` in redirect_from

**Comment by benluo** [comment]
> > For the Links > Internal check, the following failures were found:
> > 
> > * At ./_site/zh-cn/scala3/book/string-interpolation.html:2588:
> >   internally linking to /zh-cn/zh-cn/scala3/book/string-interpolation.html, which does not exist
> > 
> > don't know why?
> 
> try to remove `/zh-cn/` in redirect_from

does not work.

**Comment by jxnu-liguobin** [comment]
cc @SethTisue

File: _zh-cn/overviews/scala3-book/string-interpolation.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
+---
+title: 字符串插值
+type: chapter
+description: This page provides more information about creating strings and using 字符串插值.
+languages: zh-cn
```
Comment (bishabosha) [inline _zh-cn/overviews/scala3-book/string-interpolation.md:5]:
```suggestion
language: zh-cn
```

File: _zh-cn/overviews/scala3-book/string-interpolation.md (hunk 1 above)
Comment (bishabosha) [inline _zh-cn/overviews/scala3-book/string-interpolation.md:10]: this should not redirect from the English page
```suggestion
  - /zh-cn/overviews/core/string-interpolation.html
```

Comment (bishabosha) [reply _zh-cn/overviews/scala3-book/string-interpolation.md:10]: also _zh-cn/overviews/core/string-interpolation.md should be deleted

File: _zh-cn/overviews/scala3-book/string-interpolation.md (hunk 3 above)
Comment (bishabosha) [reply _zh-cn/overviews/scala3-book/string-interpolation.md]: without this change, the layout template treats this as the "English" page

**Review by bishabosha — APPROVED** [review]

**Comment by SethTisue** [comment]
thank you!

**Pull request merged**